### PR TITLE
Fee threshold fix

### PR DESCRIPTION
**Pull request by colinbruce**
[Ticket:#103489954](https://www.pivotaltracker.com/story/show/103489954)

### CODE REVIEW

**Comment by colinbruce** [comment]
@novotnyjakub Can you code review this please?  It's a new, urgent, bug, but coded in a more more forward thinking way.

File: spec/lib/fee_threshold_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe FeeThreshold do
+  let(:application) { create :application }
+  subject { described_class.new(application) }
+
+  context '.band' do
```
Comment (novotnyjakub) [inline spec/lib/fee_threshold_spec.rb]: This is a really really minor thing, but `.` is by convention used for class methods and `#` is used for instance methods.

File: lib/fee_threshold.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class FeeThreshold
```
Comment (novotnyjakub) [inline lib/fee_threshold.rb]: I think this is really good preemptive extraction for the refactoring we're doing :+1:

File: lib/fee_threshold.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      { lower: 1336, upper: 1665, amount: 5000 },
+      { lower: 1666, upper: 2000, amount: 6000 },
+      { lower: 2001, upper: 2330, amount: 7000 },
+      { lower: 2331, upper: 4000, amount: 8000 },
+      { lower: 4001, upper: 5000, amount: 10000 },
+      { lower: 5001, upper: 6000, amount: 12000 },
+      { lower: 6001, upper: 7000, amount: 14000 }
+    ]
+
+  def initialize(application)
```
Comment (novotnyjakub) [inline lib/fee_threshold.rb]: I'd maybe pass in the fee, rather than the whole application, so that it'd be less coupled with the `Application` model?

Comment (dotemacs) [inline lib/fee_threshold.rb]: @novotnyjakub This change is not a refactor per se, but it's rather a urgent bug fix that needs to go in. This was the bug that Jiten tried to talk to you about this morning.

**Pull request merged**